.github/workflows/overcloud-host-image-build.yml

```diff
@@ -94,11 +94,15 @@ jobs:
         if: inputs.centos
 
       - name: Upload CentOS Stream 8 overcloud host image artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: CentOS Stream 8 overcloud host image
-          path: /opt/kayobe/images/overcloud-centos-stream-8
-          retention-days: 7
+        run: |
+          kayobe playbook run \
+          src/kayobe-config/etc/kayobe/ansible/upload-image.yml \
+          -i localhost \
+          -e image_path='/opt/kayobe/images/overcloud-centos-stream-8' \
```

Comment:
I think it's a directory containing a overcloud-centos-stream-8.qcow2 image file and some other DIB artifacts.

Reply:
I've adapted the playbook to account for this. I am assuming the other DIB artifacts can be ignored, and that there is only one .qcow2 image in the directory